lost/src/main/java/com/mapzen/android/lost/internal/GeofencingApiImpl.java

```diff
@@ -74,16 +82,18 @@ public void disconnect() {
   private PendingResult<Status> addGeofence(LostApiClient client, Geofence geofence,
```

Comment:
Nice catch, that will be a nice improvement.

Tracking here: https://github.com/mapzen/lost/issues/120